Tests/TestUtilities/TestUtils.cs

```diff
@@ -155,7 +155,7 @@ public static string EnsureDefaultPropertiesFileExists(string targetDir, TestCon
 <RuleSet xmlns:xsd='http://www.w3.org/2001/XMLSchema' xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' Name='x' Description='x' ToolsVersion='14.0'>
 </RuleSet>");
 
-        private static string CreateTestSpecificFolder(TestContext testContext)
+        public static string CreateTestSpecificFolder(TestContext testContext)
```

Comment:
I don't think we need this one to be public, it looks like reinventing the wheel on more lines of code. `CreateTestSpecificFolderWithSubPaths` can do the tricks we need on less lines. See the usage here, for example:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/45df416d2d4d938a9f655d045e16e1f817bdb35b/Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/E2ETests/E2EAnalysisTests.cs#L787-L789